RecoMuon/L2MuonSeedGenerator/src/L2MuonSeedGenerator.cc

```diff
@@ -106,279 +104,298 @@ void L2MuonSeedGenerator::produce(edm::Event& iEvent, const edm::EventSetup& iSe
   L1MuonParticleCollection::const_iterator it;
   L1MuonParticleRef::key_type l1ParticleIndex = 0;
 
-  for(it = muColl->begin(); it != muColl->end(); ++it,++l1ParticleIndex) {
-    
+  for (it = muColl->begin(); it != muColl->end(); ++it, ++l1ParticleIndex) {
     const L1MuGMTExtendedCand muonCand = (*it).gmtMuonCand();
     unsigned int quality = 0;
-    bool valid_charge = false;;
+    bool valid_charge = false;
+    ;
```

Comment:
@smuzaffar @davidlange6 @fabiocos 
this is perhaps something to be added for another round of central cleanups (```;;``` -> ```;``` or drop post-clang-format empty lines terminated with a semicolon).

Reply:
@slava77 thanks for the remark. @smuzaffar @davidlange6 As far as I can see this is not strictly speaking wrong, but likely not what we would like to have, the problem is in the ";;" at the origin of this...